lib/rr/keyword_arguments.rb

```diff
@@ -10,6 +10,12 @@ def fully_supported?
           false
         end
       end
+
+      def accept_kwargs?(subject_class, method_name)
+        subject_class.instance_method(method_name).parameters.any? { |t, _| t == :key || t == :keyrest }
+      rescue NameError
```

Comment:
When is this occurred? (Methods implemented by `method_missing`?)

Reply:
> Methods implemented by `method_missing`?

Yes.

```ruby
class T < Test::Unit::TestCase
  test '' do
    obj = Object.new
    $DEBUG = true
    stub(obj).x
    obj.x
  end
end
```

```
Loaded suite t
Started
Exception `NameError' at /.../lib/rr/keyword_arguments.rb:15 - undefined method `x' for class `#<Class:#<Object:0x00007fb307a90c10>>'
Exception `NameError' at /.../lib/rr/keyword_arguments.rb:15 - undefined method `x' for class `#<Class:#<Object:0x00007fb307a90c10>>'
.
Finished in 0.002204 seconds.
```

Reply:
OK.